packages/botonic-react/src/webchat/header.jsx

```diff
@@ -26,32 +27,47 @@ const Diffuse = styled(Flex)`
   border-radius: 6px 6px 0px 0px;
 `
 
-export const DefaultHeader = props => (
-  <Diffuse color={props.color}>
-    <Flex width={1 / 4}>
-      <img
-        style={{
-          height: 24,
-          paddingTop: 15,
-          paddingLeft: 15
-        }}
-        src={Logo}
-      />
-    </Flex>
-    <Flex width={1} flexDirection='column'>
-      <HeaderTitle>Botonic</HeaderTitle>
-      <Subtitle>Online</Subtitle>
-    </Flex>
-  </Diffuse>
-)
+export const DefaultHeader = props => {
+  let HeaderImage = props.webchatState.theme.headerImage
+  let headerTitle = props.webchatState.theme.headerTitle
+  headerTitle ? headerTitle : (headerTitle = 'Botonic')
```

Comment:
is this the same as 
`headerTitle = headerTitle or 'Botonic'` ?

Reply:
yes!! thank you!!